src/test/java/io/camunda/testing/util/Utilities.java

```diff
@@ -128,4 +135,17 @@ public static void increaseTime(final ZeebeEngineClock clock, final Duration dur
     clock.increaseTime(duration);
     Thread.sleep(100);
   }
+
+  public static void completeTask(final ZeebeEngine engine, final ZeebeClient client, final String elementId) {
+    Record<JobRecordValue> lastRecord = null;
+    for (final Record<JobRecordValue> record : engine.jobRecords().withElementId(elementId)) {
+      if (record.getIntent().equals(JobIntent.CREATED)) {
+        lastRecord = record;
+      }
+    }
+    if (lastRecord != null) {
+      client.newCompleteCommand(lastRecord.getKey()).send().join();
+    }
+    waitForIdleState(engine);
```

Comment:
:thought_balloon: Maybe we can also redirect the other calls to sleep in this class to the wait method as a preparation for things to come. But need not be in this PR.